Signal/src/ViewControllers/OWSConversationSettingsViewController.m

```diff
@@ -827,10 +827,7 @@ - (void)showUpdateGroupView:(UpdateGroupMode)mode
     updateGroupViewController.conversationSettingsViewDelegate = self.conversationSettingsViewDelegate;
     updateGroupViewController.thread = (TSGroupThread *)self.thread;
     updateGroupViewController.mode = mode;
-
-    UINavigationController *navigationController =
-        [[UINavigationController alloc] initWithRootViewController:updateGroupViewController];
-    [self presentViewController:navigationController animated:YES completion:nil];
+    [self.navigationController pushViewController:updateGroupViewController animated:YES];
```

Comment:
The way I choose a transition is to consider if the new screen is more akin to "starting a new activity" or "drilling down" or "progressing" with the current activity. From a strict utility sense, they're equivalent, i.e. "going back" in a pushed view has the same end result as "canceling" a modal. As a rough heuristic (there are exceptions of course), I find that things which can be "canceled" are better expressed as modals.

I admit that this is subjective, so if I haven't persuaded you, I'm fine with this change.